prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,43 @@
+from prac_08.carv2 import Car
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose, d)rive"
+
+def main():
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    current_bill = 0
+    print("Let's Drive!")
+    print(MENU)
+    menu_choice = input(">>>>> ").lower()
+    while menu_choice != "q":
+        if menu_choice == "c":
```

Comment:
I added two extra def functions at the end to perform error checking for the user inputs for c and d (just to make sure my program wouldn't crash). Other than that, looks good and efficient.

Reply:
So, if the user enters a negative number, something that isn't a number, or tries to select a taxi number that isn't available from the taxi list, the program can identify what has happened, provide feedback to the user and ask them to re-enter their choice.